docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+**NOTE** However, this could be _any_ sort of record batch, as long as it can connect to our anchor and with the appropriate code updates!
+In other words, allow Users to use our "production" version for predictions.
+
+Some examples of why one might do this include:
+* a tool developer can ship a trained model that exemplifies expected behavior alongside "starter" training data for new user-generated and trained models.
+* A tool developer may ship a "base" model, which the User may then tune and train further in their workflow.
+* With additional UI, users may choose how/where/when to deploy models in other workflows or jobs.
+* Workflows running on Server may do **any of the above to generate serious, performant, and production-grade models**, which also
+* Empowers tool **developers AND users** to create production-grade TensorFlow AI/ML pipelines!
+
+#### Function: `on_record_batch`
+
+```python
+    def on_record_batch(self, batch: "pa.Table", anchor: "Anchor") -> None:
+        """
+        Process the passed record batch.
+
+        The method that gets called whenever the plugin receives a record batch on an input.
+
+        This method IS NOT called during update-only mode.
+
+        Parameters
+        ----------
+        batch
+            A pyarrow Table containing the received batch.
+        anchor
+            A namedtuple('Anchor', ['name', 'connection']) containing input connection identifiers.
+        """
+        if self.MODE == "PREDICT":
+            model = tf.keras.saving.load_model(f'{self.provider.tool_config["modelConfig"]["modelName"]}-exported', custom_objects={'custom_standardization': custom_standardization})
+            self.info("Loaded model, predicting...")
+            
+            try:
+                results = model.predict(batch['Beep'].to_pylist())
+            except Exception as e:
+                self.provider.io.error(f"ERR during predict")
+                raise e
+                
+            df = pd.DataFrame({"Results": [float(n[0]) for n in results]})
+            batch_to_send = pa.RecordBatch.from_pandas(df=df, preserve_index=False)
+            self.provider.write_to_anchor("Output", batch_to_send)
+            self.info("TextClassifier tool done.")
+```
+Here, we load our model with `keras. saving.load_model(...)`. Note that we append `-exported' to differentiate our training and "production" models. 
+As you develop more serious tools, you can create custom methods to deploy and store model deployment and storage. Here, we note the ability to do so!
+
+Then, we call `model.predict(...)` on our loaded model, take the results, and write them to our output anchor for use in the workflow.
+
+Now, this is all of our backend code! Since our (soon-to-exist) tests are passing, we'll move on to our front end and enable these mocked values to be dynamically set by the User!
```

Comment:
TW review:

Now, this is all of the back-end code! Since the (soon-to-exist) tests are passing, we can move on to our front end and enable these mocked values to be dynamically set by the user!